### ISSUE TITLE
`np` should be a `devDependency`, not `optional`

### ISSUE THREAD

**Issue by mogsdad** [mhipszki/eslint-formatter-summary#36]
Since `np` is currently an optional dependency, it causes unnecessary warnings for dependants using `yarn` (which has different standards for `engines` than `npm` does).

> warning np@7.6.0: The engine "git" appears to be invalid.

```json
  "optionalDependencies": {
    "np": "^7.4.0"
  }
```
The only place it is used in `eslint-formatter-summary` is in the `release` script, used only in a development environment anyway, so this should be a `devDependency` instead.

**Comment by mhipszki** [comment]
done, `2.0.0` will be published using `semantic-release`
